content/forbidden_actions.tex

```diff
@@ -88,9 +88,9 @@ \subsubsection{Penalty Area, at all times}
 If an illegal defender kicks an own goal, the goal is scored for the opponent. If there is any doubt about whether a goal should count (\eg the illegal defender infraction is called, but the robot scores the own goal immediately afterwards, before it is removed) then the decision shall be against the infringing robot.
 
 
-\subsubsection{Defender Encroachment, during Free-kick}
+\subsubsection{Defender Encroachment during Free-kick}
 
-If a robot of the offending team does enter or not attempt to leave the \FreeKickRadius area around the ball after a Free Kick (\cf \cref{sec:free_kick}) was called, ``Illegal Position'' is called. This rule does not apply for the goalkeeper robot within its own goal area. Note that the referee should not look for exact distances and rather penalize only those robots who clearly violate this rule. As a guideline, the robots of the offending team should clear the ball within \qty{10}{\second}.
+If a robot of the offending team does enter or not attempt to leave the \FreeKickRadius area around the ball after a Free Kick (\cf \cref{sec:free_kick}) was called, ``Illegal Position'' is called. Note that the referee should not look for exact distances and rather penalize only those robots who clearly violate this rule. As a guideline, the robots of the offending team should clear the ball within \qty{10}{\second}.
```

Comment:
I agree, a Free-Kick in the Penalty Box could only be the Goal-Kick or the Penalty-Kick and in both cases the goalie does not need an exception.

Reply:
This just amends the change that had already been made in the free kick section (https://github.com/RoboCup-SPL/Rules/pull/17).